blogposts/2021/release-3.30.md

```diff
@@ -0,0 +1,204 @@
+    url: https://github.com/sourcegraph/sourcegraph/pull/22756
+    category: Batch Changes
+  - description: Enabled the ability to query Batch Changes changesets, changesets stats, and file diff stats for an individual repository via the Sourcegraph GraphQL API.
+    url: https://github.com/sourcegraph/sourcegraph/pull/22744/
+    category: Batch Changes
+  - description: "Added \"Groovy\" to the initial `lang:` filter suggestions in the search bar."
+    url: https://github.com/sourcegraph/sourcegraph/pull/22755
+    category: Search
+  - description: "The `lang:` filter suggestions now show all supported, matching languages as the user types a language name."
+    url: https://github.com/sourcegraph/sourcegraph/pull/22765
+    category:
+  - description: Code Insights can now be grouped into dashboards.
+    url: https://github.com/sourcegraph/sourcegraph/issues/22215
+    category:
+  - description: "Batch Changes changesets can now be [published from the Sourcegraph UI](https://docs.sourcegraph.com/batch_changes/how-tos/publishing_changesets#within-the-ui)."
+    url: https://github.com/sourcegraph/sourcegraph/issues/18277
+    category: Batch Changes
+  - description: The repository page now has a new button to view batch change changesets created in that specific repository, with a badge indicating how many changesets are currently open.
+    url: https://github.com/sourcegraph/sourcegraph/pull/22804
+    category: Batch Changes
+  - description: "Experimental: Search-based code insights can run over all repositories on the instance. To enable, use the feature flag `\"experimentalFeatures\": { \"codeInsightsAllRepos\": true }`."
+    url: https://github.com/sourcegraph/sourcegraph/issues/22759
+    category: Search
+  - description: "Experimental: Search-based code insights can run over all repositories on the instance. To enable, use the feature flag `\"experimentalFeatures\": { \"codeInsightsAllRepos\": true }` and tick the checkbox in the insight creation/edit UI."
+    url: https://github.com/sourcegraph/sourcegraph/issues/22759
+    category: Search
+  - description: Search References is a new search sidebar section to simplify learning about the available search filters directly where they are used.
+    url: https://github.com/sourcegraph/sourcegraph/issues/21539
+    category: Search
+# Changed
+  - description: Backend Code Insights only fills historical data frames that have changed to reduce the number of searches required.
+    url: https://github.com/sourcegraph/sourcegraph/pull/22298
+    category: Search
+  - description: Backend Code Insights displays data points for a fixed 6 months period in 2 week intervals, and will carry observations forward that are missing.
+    url: https://github.com/sourcegraph/sourcegraph/pull/22298
+    category:
+  - description: Backend Code Insights now aggregate over 26 weeks instead of 6 months.
```

Comment:
@Joelkw - the previous update references 6 months. Is it 26 weeks or 6 months?

Reply:
@abschu good question! It's both – the previous update says "it will display data points for a fixed 6-month axis in 2 week intervals." This update is saying we use a more precise count of 26 weeks in place of 6 months (6 months isn't always exactly 26 weeks, which was causing flaky tests because some chunks had more/fewer weeks than others). If you click into the PR, you'll see it's just changing 6 months to 26 weeks. https://github.com/sourcegraph/sourcegraph/pull/22527/files